docs/tutorials/image_signing.md

```diff
@@ -0,0 +1,189 @@
+# How to sign and distribute container images using Podman
+
+Signing container images originates from the motivation of trusting only
+dedicated image providers to mitigate man-in-the-middle (MITM) attacks. One way
+to sign images is to utilize a GNU Privacy Guard ([GPG][0]) key. This technique
+is generally compatible with any OCI compliant container registry like
+[Quay.io][1]. It is worth mentioning that the OpenShift integrated container
+registry supports this signing mechanism out of the box.
```

Comment:
(“… making the separate sigstore unnecessary”?)

Reply:
Yes, added this note